etc/pybitmessage.profile

```diff
@@ -0,0 +1,50 @@
+# Firejail profile for pybitmessage
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/pybitmessage.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+
+noblacklist /sbin
+noblacklist /usr/local/sbin
+noblacklist /usr/sbin
```

Comment:
I'm still interested if noblacklist sbin is needes after noblacklist python was put in the right place

Reply:
Just checked. Pybitmessage doesn't working with the "noblacklist python" lines and without this "/sbin" lines. These "/sbin" lines are necessary.

Reply:
ok, thx